packages/core/src/components/popover/customModifiers.ts

```diff
@@ -21,7 +21,7 @@
 
 import type { Modifier } from "@popperjs/core";
 
-// tslint:disable object-literal-sort-keys
```

Comment:
Any clue why we're disabling sorting in this file?

Reply:
I'm guessing that was so that `matchReferenceWidthModifier` matched its definition with the reference material.